tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -277,7 +279,7 @@ def montecarlo_main_loop(
                 continue
             v_packets_energy_hist[idx] += vpackets_energy[j]
 
-    if montecarlo_configuration.VPACKET_LOGGING:
+    if virtual_packet_logging:
         for vpacket_collection in vpacket_collections:
```

Comment:
This is just for the time being- I would address the review comments in https://github.com/tardis-sn/tardis/pull/1846 and remove/edit this change accordingly.

Reply:
I believe this requires rebasing?